src/Components/PropertyInspector/PropertyTree/TreeComponents/TreeNode.tsx

```diff
@@ -24,7 +25,12 @@ const TreeNode: React.FC<NodeProps> = ({ node }) => {
                     </div>
                 )}
                 <TreeNodeMapTool node={node} />
```

Comment:
should we only show the Array tools when it's an array type? I see we hide the other control based on type here so maybe it makes sense to do teh show/hide for both at this level?

Reply:
I'm hiding the Unset here because it has "special" visibility rules for array types. I'll put a ternary operator on it.